docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -173,7 +173,7 @@ $ tsh login --user=dev --proxy=proxy.example.com:3080
 
 </TabItem>
 
-<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
+<TabItem scope={["cloud"]} label="Teleport Enterprise">
```

Comment:
We have two duplicate tab labels here. We should use "Teleport Enterprise (Self-Hosted)" and "Teleport Enterprise (Cloud)". 

There's also no need for the `scope` prop, which is left over from when users used to be able to select "Enterprise", "Self-Hosted", etc. at the top of a docs page to adjust the visibility of various components.

Reply:
Odd, I don't recall editing this hardware-key page or the others with the `TabItem` changes.